_posts/2024-04-20-16.0.0-release.md

```diff
@@ -0,0 +1,299 @@
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 16.0.0 release. This covers
+over 3 months of development work and includes [**385 resolved issues**][1]
+on [**586 distinct commits**][2] from [**119 distinct contributors**][2].
+See the [Install Page](https://arrow.apache.org/install/)
+to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 15.0.0 release, Jeffrey Vo, Jay Zhan, Bryce Mecum, Joel Lubinitsky,
+and Sarah Gilmore have been invited to be committers.
+No new members have joined the Project Management Committee (PMC).
+
+Thanks for your contributions and participation in the project!
+
+## C Data Interface notes
+
+- Added `RegisterDeviceMemoryManager` and `GetDeviceMemoryManage` for managing mappings between a device type and id to a memory manager ([GH-40698](https://github.com/apache/arrow/issues/40698)).
+- Added `RegisterCUDADevice` to register CUDA devices ([GH-40698](https://github.com/apache/arrow/issues/40698)).
+- Added `ImportFromChunkedArray` and `ExportChunkedArray` for handling Chunked Arrays in the C Stream Interface ([GH-38717](https://github.com/apache/arrow/issues/38717)).
+- Fixed an issue where string and nested types weren’t being correctly imported with DeviceArray ([GH-39769](https://github.com/apache/arrow/issues/39769)).
+- Added support for copying Arrays and RecordBatches between memory types ([GH-39771](https://github.com/apache/arrow/issues/39771)).
+
+## Arrow Flight RPC notes
+
+- Session variable RPCs were added ([GH-34865](https://github.com/apache/arrow/issues/34865))
+- Go: cookies can be copied to another connection to reuse existing credentials ([GH-39837](https://github.com/apache/arrow/issues/39837))
+- Go: enable PollFlightInfo for Flight SQL clients/servers ([GH-39574](https://github.com/apache/arrow/issues/39574))
+- Java: the JDBC driver now tries all locations the server sends it ([GH-38573](https://github.com/apache/arrow/issues/38573))
+- Java: tweak some options to give better performance ([GH-40475](https://github.com/apache/arrow/issues/40745), [GH-40039](https://github.com/apache/arrow/issues/40039))
+
+## C++ notes
+For C++ notes refer to the full changelog.
+
+## Highlights
```

Comment:
Quick note here, but heading levels should have been chosen so as to expose the logical document structure ("Highlights", "Compute"... should have been under "C++ notes").